CHANGELOG.md

```diff
@@ -26,7 +26,6 @@ Notable changes to the framework should be documented here
 	- fix use of _stdlib_path_exists_<command> in FR transport_user policy
 	  bundle (ENT-4906)
 	- partitioned __inventory table for federated reporting (ENT-4842)
-	- psql_wrapper needed full path to psql binary (ENT-4912)
```

Comment:
Actually FR was released in 3.14, so you should not take this edit, it should remain in the changelog